packages/aila/src/features/threatDetection/detectors/lakera/schema.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from "zod";
+
+const messageSchema = z.object({
+  role: z.enum(["system", "user", "assistant"]),
+  content: z.string(),
+});
+
+export const lakeraGuardRequestSchema = z.object({
+  messages: z.array(messageSchema),
+  project_id: z.string().optional(),
+  payload: z.boolean().optional(),
+  breakdown: z.boolean().optional(),
+  metadata: z.record(z.any()).optional(),
+  dev_info: z.boolean().optional(),
```

Comment:
If Lakera schema changes and we don't know about it then every request breaks, right? I wondrer if we change the `optional` to `nullish`?

Reply:
And for the schemas below?

Reply:
Unless we're confident of their schemas

Reply:
They version their schema so if it changes it would become v3 in the URL